velox/common/memory/HashStringAllocator.h

```diff
@@ -142,6 +142,8 @@ class HashStringAllocator : public StreamArena {
       return *reinterpret_cast<Header**>(end() - kContinuedPtrSize);
     }
 
+    std::string toString();
```

Comment:
Can we mark toString() a const method? Thank!

Reply:
I couldn't do that because Header::nextContinued() is not const.